falcon/bench/dj/dj/settings.py

```diff
@@ -0,0 +1,121 @@
+"""
+Django settings for dj project.
+
+Generated by 'django-admin startproject' using Django 1.11.3.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/1.11/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/1.11/ref/settings/
+"""
+
+import os
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/1.11/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 't6gj%z0ee)*xmdoqhm4^er60=s^1g_vs7y!dl2vve4-1u9^+ps'
```

Comment:
Maybe it is better to generate it on fly: 

```python 
from django.core.management import utils; 
utils.get_random_secret_key()
```
in similar way django cmd `./manage.py startapp some_app` generate *SECRET_KEY* option...